scenarioo-client/test/protractorE2E/specs/show_step.js

```diff
@@ -146,10 +150,101 @@ useCase('Show step')
         scenario('Step without HTML source attached')
             .description('If the step data contains no html source data, the HTML tab should not be displayed at all')
             .it(function () {
-
                 stepPage.goToPage('/step/Donate/find_donate_page/startSearch.jsp/0/0');
                 stepPage.assertHtmlTabIsHidden();
                 step('A step with no HTML source attached');
             });
 
+
+        scenario('ScenarioPage with comparisons')
+            .description('Navigate back and forth through the scenario steps.')
+            .it(function () {
+                 usecasePage.goToPage('/scenario/Donate/find_donate_page?branch=wikipedia-docu-example&build=2014-03-19&comparison=To%20Projectstart&tab=usecases');
+                    stepPage.clickExpandAllPagesButton();
+                    stepPage.assertFirstChangedPageDiffIconHasValue();
+                    stepPage.assertFirstChangedStepDiffIconHasValue();
+                    stepPage.assertAddedStepDiffIconTextEqualsAdded();
+                    stepPage.assertRemovedStepDiffIconTextEqualsRemoved();
+                    stepPage.assertAddedPageDiffIconTextEqualsAdded();
+                    stepPage.assertRemovedPageDiffIconTextEqualsRemoved();
+                    step('Display one scenario');
+            });
+
+        scenario('Screenshot comparison options')
```

Comment:
but dont worry, you can keep it and we can improve it later. Maybe would be a good session once to improve our e2e tests structure into real "use cases" and "scenarios".